CHANGELOG.md

```diff
@@ -1,5 +1,201 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.38
+
+<!-- generated comparing v5.0.0-alpha.37..next -->
+
+_Jun 23, 2021_
+
+Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 We have completed all the [breaking changes](https://github.com/mui-org/material-ui/issues/20012). The plan to release 5.0.0-beta.0 is on July 1st and will start to promote its usage over v4.
+- 🎨 We have updated `Slider` to match current [material design spec](https://material.io/components/sliders).
```

Comment:
```suggestion
- 🎨 We have updated `Slider` to match current [Material Design guidelines](https://material.io/components/sliders).
```

I think we're now using "guidelines" instead of "spec"